daffodil-test-ibm1/src/test/scala/org/apache/daffodil/TresysTests.scala

```diff
@@ -123,40 +123,33 @@ class TresysTests {
   @Test def test_multiple_diagnostics2() { runnerMD.runOneTest("manyErrors1") }
   @Test def test_multiple_diagnostics3() { runnerMD_NV.runOneTest("manyErrors2") }
 
-  // not found. Debug later.
-  // @Test def test_duplicateDefineFormatsOneSchema() { runnerMD.runOneTest("duplicateDefineFormatsOneSchema") }
-
   @Test def test_nested_separator_delimited_baseline() { runnerNSD.runOneTest("baseline") }
 
   // Fails in IBM DFDL - ambiguous separator/terminator not accepted.
   @Test def test_nested_separator_delimited_baseline_ibm() { runnerNSD.runOneTest("baseline_ibm") }
 
   @Test def test_nested_separator_delimited_basicNest() { runnerNSD.runOneTest("basicNest") }
-  // Fails infinite loop
-  // @Test def test_nested_separator_delimited_basicNest2() { runnerNSD.runOneTest("basicNest2")}
+  @Test def test_nested_separator_delimited_basicNest2() { runnerNSD.runOneTest("basicNest2")}
 
-  // Fails, index out of bounds
+  // Nested delimiter issues - DAFFODIL-2101
   // @Test def test_nested_separator_delimited_nest1() { runnerNSD.runOneTest("nest1")}
-  // Fails infinite loop
+  // Nested delimiter issues - DAFFODIL-2101
   // @Test def test_nested_separator_delimited_nest2() { runnerNSD.runOneTest("nest2")}
-  // Fails infinite loop
-  // @Test def test_nested_separator_delimited_nest3() { runnerNSD.runOneTest("nest3")}
+  @Test def test_nested_separator_delimited_nest3() { runnerNSD.runOneTest("nest3")}
 
   /* Very big test data files, so each is in its own TDML file */
 
-  //  @Test def test_AB007() { runnerAB7.runOneTest("AB007") }
-  //  @Test def test_AB008() { runnerAB8.runOneTest("AB008") }
-  //  @Test def test_AB009() { runnerAB9.runOneTest("AB009") }
```

Comment:
Looks like tests AB008 and AB009 were removed?

Reply:
It seems that they were removed before.  There doesn't appear to be a TDML file for either of these tests anymore.

Reply:
Long way back, but I think there were just too many variations in data volume for what was the same test really, so we removed some of them.